CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Change Log
 
+## 0.4.16
+
+**released 2022-05-27**
+
+### Added
+
+* Add option to configure timeouts in loadbalancer  [#123](https://github.com/freiheit-com/kuberpult/pull/156)
```

Comment:
Please also add removed the option to change PVC storage size https://github.com/freiheit-com/kuberpult/pull/155

Reply:
done